### PR TITLE
add foldername kwarg to servedocs

### PR DESCRIPTION
**Pull request by cserteGT3**
As discussed in #99. For documentation: is the record in the docstrings enough?
For tests I shamelessly copied the testsets and modified them, hope it's fine.

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -10,9 +10,10 @@ triggered to regenerate the documents, subsequently the LiveServer will render t
 in `docs/build`.
 """
```
Comment (tlienart) [inline src/utils.jl:11]: Please fix the function signature & add a short note about folder name being the name of the folder if not docs

File: src/utils.jl [hunk 2]
```diff
@@ -117,9 +118,11 @@ assumed that they are in `docs/src`.
 * `doc_env=false` is a boolean switch to make the server start by activating the doc environment or not (i.e. the `Project.toml` in `docs/`).
 * `skip_dir=""` is a subpath of `docs/` where modifications should not trigger the generation of the docs, this is useful for instance if you're using Weave and Weave generates some files in `docs/src/examples` in which case you should give `skip_dir=joinpath("docs","src","examples")`.
 * `skip_dirs=[]` same as `skip_dir`  but for a vector of such dirs. Takes precedence over `skip_dir`.
+* `foldername="docs"` use `foldername` instead of "docs" everywhere.
```
Comment (tlienart) [inline src/utils.jl:121]: (use ... everywhere) --> `specify the name of the content folder if different than `"docs"`.

**Comment by tlienart** [comment]
This looks great, many thanks!

Could you please lightly update the docs? (`man/functionalities/`) the additional keyword sections

Then I'll merge and do a minor release with it, thanks!

File: src/utils.jl (hunk 1 above)
Comment (cserteGT3) [reply src/utils.jl:11]: My bad, forgot the signatures.

**Comment by tlienart** [comment]
Thanks Tamás!

**Pull request merged**